### ISSUE TITLE
Add cdo to the Software page

### ISSUE THREAD

**Issue by jypeter** [cf-convention/cf-convention.github.io#360]
I have just noticed that the [Software that “Understands” CF Data](http://cfconventions.org/software.html) page does not mention [Climate Data Operators (cdo)](https://code.mpimet.mpg.de/projects/cdo)

This should surely be added ! :-)

**Comment by erget** [comment]
I agree! @jypeter would you be interested in sending a PR with the mention? :)
